src/Components/Components/src/RenderTree/Renderer.cs

```diff
@@ -770,6 +813,7 @@ private void ProcessRenderQueue()
 
         _isBatchInProgress = true;
         var updateDisplayTask = Task.CompletedTask;
+        var batchStartTimestamp = ComponentMetrics != null && ComponentMetrics.IsBatchEnabled ? Stopwatch.GetTimestamp() : 0;
```

Comment:
Don't remember the exact syntax, but you can write it like below

```suggestion
        var batchStartTimestamp = ComponentMetrics  is { IsBatchEnabled: true } ? Stopwatch.GetTimestamp() : 0;
```